docs/rules/no-unnecessary-reactive-curlies.md

```diff
@@ -0,0 +1,48 @@
+---
+pageClass: "rule-details"
+sidebarDepth: 0
+title: "svelte/no-unnecessary-reactive-curlies"
+description: "disallow unnecessary curly braces around reactive statements"
+since: "v2.4.0"
+---
+
+# svelte/no-unnecessary-reactive-curlies
+
+> disallow unnecessary curly braces around reactive statements
+
+## :book: Rule Details
+
+This rule reports if curly brackets (`{` and `}`) are used unnecessarily around a reactive statement body containing only a single expression.
+
+<ESLintCodeBlock>
+
+<!--eslint-skip-->
+
+```svelte
+<script>
+  /* eslint svelte/no-unnecessary-reactive-curlies: "error" */
+  
+  /* ✓ GOOD */
+  $: foo = "red";
+
+  /* ✗ BAD */
+  $: {
+    foo = "red";
+  }
+</script>
+```
+
+</ESLintCodeBlock>
+
+## :wrench: Options
+
+Nothing.
+
```

Comment:
What do you think about adding something like the following wording to help with the migration?

> ## :heart: Compatibility
> 
> This rule was taken from [@tivac/eslint-plugin-svelte].  
> This rule is compatible with `@tivac/svelte/reactive-curlies` rule.
> 
> [@tivac/eslint-plugin-svelte]: https://github.com/tivac/eslint-plugin-svelte/

```suggestion

## :heart: Compatibility

This rule was taken from [@tivac/eslint-plugin-svelte].  
This rule is compatible with `@tivac/svelte/reactive-curlies` rule.

[@tivac/eslint-plugin-svelte]: https://github.com/tivac/eslint-plugin-svelte/

```